modules/machineset-yaml-aws.adoc

```diff
@@ -136,18 +140,18 @@ ifndef::edge[]
                   - <infrastructure_id>-private-<zone> <8>
 endif::edge[]
 ifdef::edge[]
-              id: <value_of_PublicSubnetIds> <8>
+            id: <value_of_PublicSubnetIds> <8>
           publicIp: true
 endif::edge[]
-          tags:
-            - name: kubernetes.io/cluster/<infrastructure_id> <1>
+          tags: <9>
```

Comment:
I moved the callout for `tags` to the top level to make it easier to talk about this section more generally